src/main/java/net/sacredlabyrinth/phaed/simpleclans/hooks/discord/exceptions/DiscordHookException.java

```diff
@@ -0,0 +1,19 @@
+package net.sacredlabyrinth.phaed.simpleclans.hooks.discord.exceptions;
+
+import net.sacredlabyrinth.phaed.simpleclans.SimpleClans;
+import org.jetbrains.annotations.NotNull;
+
+import static net.sacredlabyrinth.phaed.simpleclans.managers.SettingsManager.ConfigField.DEBUG;
+
+public class DiscordHookException extends Exception {
+    public DiscordHookException() {
+        super();
+    }
+
+    public DiscordHookException(String message, @NotNull Object... args) {
+        super(String.format(message, args));
+        if (SimpleClans.getInstance().getSettingsManager().is(DEBUG)) {
+            SimpleClans.getInstance().getLogger().warning(String.format(message, args));
```

Comment:
It's unusual for a exception warning to be printed in the constructor

Reply:
> It's unusual for an exception warning to be printed in the constructor.

I can agree with it, but isn't it free our code base? 🤔

Reply:
Can you rephrase that question?

Reply:
With the constructor the code will be shorter. 👀

Reply:
Not really.
SimpleClans has a debug (in the main class) method, if used in a try/catch it would be even shorter.